cl_dll/vgui_ScorePanel.cpp

```diff
@@ -268,6 +276,9 @@ ScorePanel::~ScorePanel()
 #endif
 	if (m_pFlagIcon)
 		delete m_pFlagIcon;
+
+	m_TitleLabel.setImage(nullptr);
+	delete m_pTitleImage;
```

Comment:
Shouldn't this be guarded by shouldFallback?

Reply:
Not really. `vgui::Label` has no image set by default and deleting a null pointer is completely safe.

Reply:
I don't see `m_pTitleImage` initialized to `nullptr` anywhere?

Reply:
That's my mistake. I initially was going to always construct UnicodeTextImage and changed the code later. That would actually be a really subtle bug since most of the time `shouldFallback` would return false (and also, ScorePanel is never destroyed).